proposal.md

```diff
@@ -7,14 +7,25 @@ Name: ____________           ID:   ____________
 > how it will work. I don't need more than three or four sentences,
 > unless you feel compelled to write more.
 
+My application will be a homework tracking app for students. The
+purpose of the application is to help students keep track of their
+homework and exams. Students will be able to add homework assignments
+and exams to the application and clissify them by course and add due
+dates. The application will then organize all of the information and
+display it to the student in a manner which helps them stay organized
+and complete assignments on time.
+
 ## Outline Structure
 
 > Describe how you'll organize your code. What is the process and
 > supervision structure? If it uses a framework, how does it fit in. I
 > just need to understand the direction you plan to take. Again, three
 > or four sentences are probably enough.
 
-
-
-> replace all the ">" lines with your content, then push this to
-> github and issue a merge request.
+My application will have a Phoenix framework web interface for
```

Comment:
Will the database access be part of the Phoenix app, or will it be a free-standing application?

Reply:
The database access part will be a free-standing application, because as you recommended in class, the front-end and back-end logic should be separate and de-coupled.